src/Pages/Login.js

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
-import { useHistory } from 'react-router-dom';
+import Axios from "axios";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
 
 export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
 
-	const history = useHistory()
-	
-	function login() {
-		localStorage.setItem("isAuthenticated", true)
-		history.replace("/dashboard")
-	}
+  const history = useHistory();
 
+  //this could be changed to an async function
+  //this verification route will work if you run the server from index.js on your local machine
+  //currently the credentials username:Admin password:password will work
+  //a registration route is set up under POST "http://localhost:5000/authentication/register" (also accepts username and pw)
+  function login() {
+    Axios({
+      method: "POST",
+      data: {
+        username: username,
+        password: password,
+      },
+      withCredentials: true,
+      url: "http://localhost:5000/authenticate/login",
+    }).then((res) => {
+      if (res.data === "Successfully Authenticated") {
+        setUsername("");
+        setPassword("");
+        localStorage.setItem("isAuthenticated", true);
+        history.replace("/dashboard");
+      }
```

Comment:
Do you maybe want to add an error handling after your if statement (line30)? So that if the login does not work, there is a message to the user saying something like "hey dude, there was an error, please try again". 
<img width="534" alt="Screen Shot 2020-11-07 at 18 17 53" src="https://user-images.githubusercontent.com/58600732/98447574-3107ca00-2126-11eb-8d97-55fed48c2999.png">